src/components/authorization/authorization.service.ts

```diff
@@ -187,6 +194,10 @@ export class AuthorizationService {
   }
 
   async roleAddedToUser(id: string, roles: Role[]) {
+    await this.afterTransaction(() => this.doRoleAddedToUser(id, roles));
```

Comment:
So is the thing that's changing is that adding roles to users is being retried?

Reply:
Yeah but mostly to change to run after the current transaction finishes.